CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.1.10 (2024-04-12)
```

Comment:
@chrisguillory do you think this should be a minor version update rather than patch? (v0.2.0)?

Reply:
I think that makes sense, especially according to these links -- these are new features.
* https://app.stainlessapi.com/docs/guides/publish#versioning-and-releases
* https://semver.org/